lib/ansible/modules/remote_management/oneview/oneview_logical_interconnect_group.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python
+# Copyright (c) 2016-2017 Hewlett Packard Enterprise Development LP
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: oneview_logical_interconnect_group
+short_description: Manage OneView Logical Interconnect Group resources
+description:
+    - Provides an interface to manage Logical Interconnect Group resources. Can create, update, or delete.
+version_added: "2.5"
+requirements:
+    - hpOneView >= 4.0.0
+author:
+    - Felipe Bulsoni (@fgbulsoni)
+    - Thiago Miotto (@tmiotto)
+    - Adriane Cardozo (@adriane-cardozo)
+options:
+    state:
+        description:
+            - Indicates the desired state for the Logical Interconnect Group resource.
+              C(present) will ensure data properties are compliant with OneView.
+              C(absent) will remove the resource from OneView, if it exists.
+        default: present
+        choices: ['present', 'absent']
+    data:
+        description:
+            - List with the Logical Interconnect Group properties.
+        required: true
+extends_documentation_fragment:
+    - oneview
+    - oneview.validateetag
+'''
+
+EXAMPLES = '''
+- name: Ensure that the Logical Interconnect Group is present
+  oneview_logical_interconnect_group:
+    config: /etc/oneview/oneview_config.json
+    state: present
+    data:
+      name: 'Test Logical Interconnect Group'
+      uplinkSets: []
+      enclosureType: 'C7000'
+      interconnectMapTemplate:
+        interconnectMapEntryTemplates:
+          - logicalDownlinkUri: ~
+            logicalLocation:
+                locationEntries:
+                    - relativeValue: "1"
+                      type: "Bay"
+                    - relativeValue: 1
+                      type: "Enclosure"
+            permittedInterconnectTypeName: 'HP VC Flex-10/10D Module'
+            # Alternatively you can inform permittedInterconnectTypeUri
```

Comment:
Are you sure this works with the inline comment ?

Reply:
Yep, commented stuff just gets ignored when running, I often use it to have a few different values which I switch in between.
In this specific case I've left the comment in the example because in case the user does not read the module documentation, at least he/she will know that this field exists as it is not default from the REST API documentation.